src/udiskslinuxencrypted.c

```diff
@@ -788,7 +788,7 @@ handle_change_passphrase (UDisksEncrypted        *encrypted,
   const gchar *action_id;
   GError *error = NULL;
   gchar *device = NULL;
-  CryptoJobData data = { NULL, NULL, NULL, NULL, NULL, 0, FALSE, FALSE };
+  CryptoJobData data = { NULL, NULL, NULL, NULL, NULL, 0, 0, FALSE, FALSE, FALSE, NULL };
```

Comment:
How come so many values are added here while there's only `type` added to the struct type below?

Reply:
It was incorrect before and no one noticed (and gcc started complaining when I added only one new `NULL`).

Reply:
What about `CryptoJobData data = { 0, };` ?

Reply:
I'm not sure if this works with clang, see https://github.com/storaged-project/libblockdev/pull/325

Reply:
Btw. the storaged-project/libblockdev#325 was about various forms of `{0}` but not `{0,}` - the comma makes a big difference. I'll try to get old clang here and test this when I have some free time. FTR, glib itself contains number of such initializers.